play-ebean/src/main/java/play/db/ebean/EBeanComponents.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+package play.db.ebean;
+
+
+import play.Configuration;
+import play.Environment;
+import play.api.db.evolutions.DynamicEvolutions;
+import play.db.DBApi;
+import play.inject.ApplicationLifecycle;
+
+/**
+ * Classes for Java compile time dependency injection.
+ */
+public interface EBeanComponents {
+
+    ApplicationLifecycle applicationLifecycle();
+
+    Environment environment();
+
+    Configuration configuration();
```

Comment:
`ConfigurationComponents` provides an implementation. This is just declaring a dependency.